api/rest/v1/bulk_observations_series.md

```diff
@@ -0,0 +1,341 @@
+---
+layout: default
+title: Get a series of observations
+nav_exclude: true
+parent: v1 REST
+grand_parent: API
+published: false
+permalink: /api/rest/v1/bulk/observations/series
+ 
+---
+ 
+# /v1/bulk/observations/series
+
+Retrieve all values for multiple variables and entities.
```

Comment:
all values sounds a bit too general, maybe "all values" => "observation series"? 

Here, the tips below, and **_point.md all mention "single point" and "all values". Might make this more explicit by saying "observation at a date" vs "observation time series" throughout consistently.